YLTableViewTests/YLTableViewDataSourceTests.m

```diff
@@ -0,0 +1,94 @@
+//
+//  YLTableViewDataSourceTests.m
+//  YLTableView
+//
+//  Created by Ushhud Khalid on 12/9/15.
+//  Copyright © 2015 Yelp. All rights reserved.
+//
+
+#import "YLTableView.h"
+#import "YLTableViewBaseTestCase.h"
+#import "YLTableViewCell.h"
+#import "YLTableViewCellEstimatedRowHeight.h"
+#import "YLTableViewCellTestStub.h"
+#import "YLTableViewDataSource.h"
+#import "YLTableViewDataSourceTestStub.h"
+
+@interface YLTableViewDataSourceTests : YLTableViewBaseTestCase
+
+@property (strong, nonatomic) YLTableView *tableView;
+@property (strong, nonatomic) YLTableViewDataSourceTestStub *dataSource;
+
+@property (strong, nonatomic) NSIndexPath *indexPathOfConformingCell;
+@property (strong, nonatomic) NSIndexPath *indexPathOfNonConformingCell;
+
+@end
+
+@implementation YLTableViewDataSourceTests
+
+#pragma mark - Setup/Teardown
+
+-(void)setUp {
+  [super setUp];
+
+  _indexPathOfConformingCell = [NSIndexPath indexPathForRow:0 inSection:0];
```

Comment:
seems like for these, you could just have a global const section, and then 2 global const rows to avoid having to carry state through your tests